tools/make/lib/lint/javascript/eslint.mk

```diff
@@ -44,6 +44,9 @@ ESLINT_CONF_BENCHMARKS ?= $(CONFIG_DIR)/eslint/.eslintrc.benchmarks.js
 # Define the path to the ESLint ignore file:
 ESLINT_IGNORE ?= $(ROOT_DIR)/.eslintignore
 
+# Define optional path for storing lint failure reports:
+ESLINT_ERROR_LOG ?=
+
 # Define the command-line options to use when invoking the ESLint executable:
 ESLINT_FLAGS ?= \
```

Comment:
I would suggest converting this to an internal variable `eslint_flags` and then creating a separate `ESLINT_FLAGS ?=` which is user defined. Then you can concat the two variables and use `eslint_flags` below.

We do this elsewhere in various make recipes.